### ISSUE TITLE
[Bug]: Cloud9 environment was not created

### ISSUE THREAD

**Issue by tejashreeb12** [aws-samples/eks-workshop-v2#399]
### Installation method

Own AWS account

### What happened?

I followed all the steps to install Terraform and apply to create infrastructure. I am on a Mac and using my Isengard account to run this Lab. I am struck at the step to use IDE for further commends. 

### What did you expect to happen?

After running Setup, When I opened Cloud9, I should see an environment already created for EKS workshop

### How can we reproduce it?

BY following the setup steps

### EKS version

I did not reach to the step in IDE to find out EKS version

**Comment by tejashreeb12** [comment]
added 
connection_type = "CONNECT_SSM" to  modules/ide/cloud.tf

it worked

**Comment by niallthomson** [comment]
Hi @tejashreeb12. It looks like you got past this but it would be great to get a little more information.

Do you run the Terraform with the same role as you access the AWS console with?

**Comment by KgomotsoWelcome** [comment]
I am running into the same issue. I can see that the Cloud9 environment was created as I checked CloudTrail and saw the API call made and I am also able to see the underlying EC2 instance in the list of EC2 instances, however, in the Cloud9 console, I cannot see the see Cloud9 environment. 

In my case, I used a Cloud9 instance to run the initial commands. The role used by Terraform is arn:aws:iam::<account-ID>:user/admin. And to check the instance using the AWS console, I access the console using an assumed role
